standalone-metastore/metastore-common/src/main/java/org/apache/hadoop/hive/metastore/IMetaStoreClient.java

```diff
@@ -1545,83 +1545,6 @@ GetPartitionsPsWithAuthResponse listPartitionsWithAuthInfoRequest(GetPartitionsP
   PartitionsResponse getPartitionsRequest(PartitionsRequest req)
           throws NoSuchObjectException, MetaException, TException;
 
-  /**
-   * Get partitions by a list of partition names.
-   * @param db_name database name
-   * @param tbl_name table name
-   * @param part_names list of partition names
-   * @param getColStats if true include statistics in the Partition object
-   * @param engine engine sending the request
-   * @return list of Partition objects
-   * @throws NoSuchObjectException No such partitionscatName
-   * @throws MetaException error accessing the RDBMS.
-   * @throws TException thrift transport error
-   */
-  List<Partition> getPartitionsByNames(String db_name, String tbl_name, List<String> part_names,
-      boolean getColStats, String engine) throws NoSuchObjectException, MetaException, TException;
-
-  /**
-   * Get partitions by a list of partition names.
-   * @param catName catalog name
-   * @param db_name database name
-   * @param tbl_name table name
-   * @param part_names list of partition names
-   * @return list of Partition objects
-   * @throws NoSuchObjectException No such partitions
-   * @throws MetaException error accessing the RDBMS.
-   * @throws TException thrift transport error
-   */
-  List<Partition> getPartitionsByNames(String catName, String db_name, String tbl_name,
```

Comment:
Seems this method is realeased from version 3.0.0

Reply:
Thanks for noticing. Added back with depreaction